src/version_info.rs

```diff
@@ -92,64 +71,4 @@ mod tests {
             result => panic!("unexpected error: {:?}", result),
         }
     }
-
-    #[test]
-    pub fn get_version_info_pre_0_15_0() -> Result<(), ContractError> {
-        let mut deps = mock_dependencies(&[]);
-
-        #[allow(deprecated)]
-        set_legacy_contract_info(
-            &mut deps.storage,
-            &contract_info::ContractInfo {
-                name: "contract_name".into(),
-                definition: "contract_def".to_string(),
-                version: "0.14.99".to_string(),
-                bind_name: "bind_name".to_string(),
-                base_denom: "base_1".to_string(),
-                convertible_base_denoms: vec![],
-                supported_quote_denoms: vec![],
-                executors: vec![],
-                issuers: vec![],
-                ask_required_attributes: vec![],
-                bid_required_attributes: vec![],
-                price_precision: Uint128::new(2),
-                size_increment: Uint128::new(100),
-            },
-        )?;
-
-        let version_info = get_version_info(&deps.storage);
-        match version_info {
-            Ok(version_info) => {
-                assert_eq!(version_info.definition, "contract_def");
-                assert_eq!(version_info.version, "0.14.99");
-            }
-            result => panic!("unexpected error: {:?}", result),
-        }
-
-        Ok(())
-    }
-
-    #[test]
-    pub fn get_version_info_unknown() -> Result<(), ContractError> {
```

Comment:
why remove this? we could still check that a `ContractError` was returned i think